src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes"),
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Paused").withText("shirts")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+
+        assertThat(kenshooKeywords, hasSize(2));
+    }
+
+    @Test
+    public void testDownloadActiveGoogleKeyword(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+        assertThat(kenshooKeywords.get(0), isKeyword(withIdInTarget(1), withStatus("Active"), withText("shoes")));
+    }
+
+    private Matcher<KenshooKeyword> withIdInTarget(Integer idInTarget) {
+        return new ArgumentMatcher<KenshooKeyword>() {
+            @Override
+            public boolean matches(Object o) {
+                return ((KenshooKeyword) o).idInTarget ==  idInTarget;
+            }
+        };
+    }
+
```

Comment:
Well done!
I'm happy you master the Matcher technique.

It looks like a burden writing a matcher for every field of the keyword, right...?
We could be a little more lazy and write a test like:
```
Keyword downloadedKeyword = getFirst(downloadKeyword(using(selector().withCapaignId(1000))));
assertThat(downloadedKeyword.status, equalTo("Active"));
```

without using a matcher.
It is actually my fault for requiring matchers without a real need for them... ooops.